synapse/models/transport.py

```diff
@@ -8,6 +8,15 @@ def getModelDefs(self):
                 ('transport:direction', ('hugenum', {'modulo': 360}), {
                     'doc': 'A direction measured in degrees with 0.0 being true North.'}),
 
+                ('transport:land:vehicle', ('guid', {}), {
+                    'doc': 'An individual vehicle.'}),
+
+                ('transport:land:registration', ('guid', {}), {
+                    'doc': 'Registration issued to a contact for a land vehicle.'}),
+
+                ('transport:land:license', ('guid', {}), {
+                    'doc': 'A license to operate a land vehicle issued to a contact.'}),
```

Comment:
air licenses too?

Reply:
eventually, sure. out of scope for this PR.